examples/tutorial_udp.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""Example for sending batch information to InfluxDB via UDP."""
+
+"""
+INFO: In order to use UDP, one should enable the UDP service from the
+`influxdb.conf` under section
+    [[udp]]
+        enabled = true
+        bind-address = ":8089" # port number for sending data via UDP
```

Comment:
I think this value should be consistent with the port used in the default example code.

Reply:
The UDP port by default configuration is 8089. Port 8086 is required to connect to the DB.